src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs

```diff
@@ -711,6 +711,120 @@
   {:path (str "/billing/" project-id "/" role "/" user-email)
    :method :delete})
 
+(defn get-library-attributes [on-done]
+  (utils/ajax-orch
+    "/library-attributedefinitions-v1"
+    {:method :get
+     :on-done on-done
+     :canned-response
```

Comment:
I don't love the idea of having to run another service, but it may be time to use Mock Server and move away from our own implementation.